deploy/clusterrole.yaml

```diff
@@ -36,6 +36,9 @@ rules:
 - apiGroups: ["config.openshift.io"]
   resources: ["infrastructures"]
   verbs: ["get", "list", "watch"]
+- apiGroups: ["apiextensions.k8s.io"]
+  resources: ["customresourcedefinitions"]
```

Comment:
I think we should add this one also to the config folder this way when we generate the csv file for OLM it should be also in place

Reply:
Added, though I think we still need some cleanup in that folder.